admission-webhook/utils_test.go

```diff
@@ -93,3 +99,52 @@ func shuffleContainers(a []corev1.Container) {
 		a[i] = tmp
 	}
 }
+
+func GenerateTestCertAndKey() {
```

Comment:
will this run if the files are already there?  should we remove the existing files and re-create them?

Reply:
Add CleanTestdata and call it after running all unit tests.